packages/connector-jsdom/src/resource-loader.ts

```diff
@@ -19,14 +19,14 @@ export default class CustomResourceLoader extends ResourceLoader {
         this._connector = connector;
     }
 
-    public fetch(url: string, options: { element: HTMLElement }) {
+    public async fetch(url: string, options: { element: HTMLElement }): Promise<Buffer | null> {
```

Comment:
It cleans up the `return null` further down in the function to actually return a `Promise<null>`  which was my interpretation of the intent (please correct me if I'm wrong):
https://github.com/webhintio/hint/blob/482a09d557f28e22853a37603f77265443bb0a96/packages/connector-jsdom/src/resource-loader.ts#L43

I could have explicitly returned a new `Promise`, but this felt like a more complete fix to avoid having an unwrapped `null` returned from the function accidentally in the future.

Reply:
ohhh, I see, you are rigth, that was an error.
Anyway, you don't need to return a new promise you can just `return Promise.resolve()`